src/wallet/wallet2.cpp

```diff
@@ -5297,6 +5297,95 @@ bool wallet2::parse_uri(const std::string &uri, std::string &address, std::strin
   return true;
 }
 //----------------------------------------------------------------------------------------------------
+uint64_t wallet2::get_blockchain_height_by_date(uint16_t year, uint8_t month, uint8_t day)
+{
+  uint32_t version;
+  if (!check_connection(&version))
+  {
+    throw std::runtime_error("failed to connect to daemon: " + get_daemon_address());
+  }
+  if (version < 0x1006)
```

Comment:
10006 too. Maybe add a MAKE_RPC_VERSION(x,y) (((x)<<16)|(y)), and make CORE_RPC_VERSION use this. This will prevent bugs like this.